README.md

```diff
@@ -119,7 +119,7 @@ version,
 opam source --dev-repo --pin lwt
 ```
 
-This will also install the development dependency OASIS.
+This will also install the development dependency jbuilder.
```

Comment:
`jbuilder` isn't a development-only dependency like OASIS is/was, so we can remove this entire sentence. The current `README.md` has a completely different contributing section anyway, though.